### ISSUE TITLE
JAWS and VO highlight random area in sim when reading about sliders

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/ohms-law#147]
From https://github.com/phetsims/QA/issues/327.  JAWS highlights large and seemingly unrelated area in the sim when reading content. This large highlight is also seen in Mac VO.

https://drive.google.com/file/d/116L6UpV235rqAKw6ILUWwsoO_X7LCbL4/view?usp=sharing

**Comment by jessegreenberg** [comment]
I am going to see if I can produce with JAWS on my end. It is likely due to the way the PDOM is styled for mobile accessibility.

**Comment by jessegreenberg** [comment]
Ah, I am seeing this now after updating to JAWS 2019.

**Comment by jessegreenberg** [comment]
JAWS 2019 now highlights text and other things on the screen as they are read. In this case, it is trying to highlight the unordered list that contains the sliders. This is what this bug looks like in the context of ohms-law while it reads "Slider Controls, list of Two Items":
![image](https://user-images.githubusercontent.com/6396244/58651265-d0dce980-82de-11e9-8129-cd42dd536424.png)

I tried to see what it would look like in another context, and the highlight was messed up in a vanilla HTML page too:
![image](https://user-images.githubusercontent.com/6396244/58651160-878c9a00-82de-11e9-97c7-22386e97f207.png)

The sim image is undoubtedly distracting, but I don't now whether we should spend time on this given that this JAWS feature doesn't work very well yet with web content. Time spent will likely go to finding a workaround that makes the highlight smaller, but it is not likely that this can be fixed entirely. @terracoda and @emily-phet can you please comment on whether we should work on this?

**Comment by terracoda** [comment]
VoiceOver also has it's own highlighting which we cannot control.
I agree @jessegreenberg, I don't think we should spend time on this. JAWS may change this feature.

**Comment by jessegreenberg** [comment]
OK, thanks for the input @terracoda, sounds good. Removing the `ul` in #148 may also remove the highlight in this case.

**Comment by jessegreenberg** [comment]
Indeed, #148 made the rectangle go away in this case.
